Dockerfile

```diff
@@ -37,15 +37,18 @@ RUN export DATABASE_URL=mysql2://localhost/temp && \
     apt-get install -y mysql-server && \
     /etc/init.d/mysql start && \
     cp config/app_config.yml.SAMPLE config/app_config.yml && \
+    cp config/database.yml.MySQL_SAMPLE config/database.yml && \
     bundle exec rake db:setup assets:precompile && \
     rm -Rf config/app_config.yml tmp/* && \
     /etc/init.d/mysql stop && \
     rm -Rf /run/mysqld /tmp/* /var/tmp/* /var/lib/mysql /var/log/mysql* && \
     apt-get purge -y --auto-remove mysql-server && \
     rm -Rf /var/lib/apt/lists/* /var/cache/apt/*
 
-# Make relevant dirs writable for app user
+# Make relevant dirs and files writable for app user
 RUN mkdir -p tmp && \
+    chown nobody config/app_config.yml && \
+    chown nobody db/schema.rb && \
```

Comment:
Hmmm when did you run into this not being writable?

Reply:
all the time when i do migrations. if i migrate a singe coop it's ok since the error is at the end, but when i want to migrate multiple coops it stops after the first one